helm-chart/templates/analytics/clickhouse/initializer/job.yml

```diff
@@ -0,0 +1,56 @@
+{{- if .Values.analytics.clickhouse.initializer.enabled }}
+{{- $localValues := .Values.analytics.clickhouse.initializer -}}
+
+apiVersion: batch/v1
+kind: Job
+metadata:
+  # Jobs are mostly immutable, so Helm can't update them when the user upgrades
+  # a release. Work around this by including the revision in the job name, so that
+  # every upgrade creates a new job.
+  # See also <https://github.com/helm/helm/issues/7082>.
+  name: {{ .Release.Name }}-clickhouse-initializer-r{{ .Release.Revision }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "cvat.labels" . | nindent 4 }}
+    app: cvat-app
+    tier: clickhouse
+    component: initializer
+spec:
+  template:
+    metadata:
+      labels:
+        {{- include "cvat.labels" . | nindent 8 }}
+        app: cvat-app
+        tier: clickhouse
+        component: initializer
+    spec:
+      containers:
+        - name: clickhouse
+          image: {{ .Values.clickhouse.image.registry }}/{{ .Values.clickhouse.image.repository }}:{{ .Values.clickhouse.image.tag }}
+          imagePullPolicy: Never
```

Comment:
```suggestion
          imagePullPolicy: IfNotPresent
```

Reply:
OTOH, clickhouse-client versions are likely to be compatible with a wide range of clickhouse versions, so it may be overcautious.